commands/kubernetes.go

```diff
@@ -1690,6 +1694,11 @@ func buildClusterUpdateRequestFromArgs(c *CmdConfig, r *godo.KubernetesClusterUp
 	}
 	r.SurgeUpgrade = surgeUpgrade
 
+	ha, err := c.Doit.GetBool(c.NS, doctl.ArgHA)
```

Comment:
I think this will have to become `GetBoolPtr` given https://github.com/digitalocean/godo/pull/565#discussion_r993379217.